helm-frontend/src/layouts/Main.tsx

```diff
@@ -1,10 +1,11 @@
 import { Outlet } from "react-router-dom";
-import Nav from "@/components/NavBar/NavBar";
+import NavBar from "@/components/NavBar/NavBar";
+import MinimalNavBar from "@/components/MinimalNavBar";
 
 export default function Main() {
   return (
     <>
-      <Nav />
+      {window.PROJECT_ID === "global" ? <MinimalNavBar /> : <NavBar />}
```

Comment:
Alternatively can make this boolean take place in NavBar, but this causes the test to act weird and also this allows us to have more decoupled components for the global landing page, in case we want to fully separate it.

Reply:
Seems fine.